views/plugin-page.php

```diff
@@ -80,6 +81,7 @@
 
 	// Display the plugin intro text (can be replaced with custom text through the filter below).
 	echo wp_kses_post( apply_filters( 'pt-ocdi/plugin_intro_text', $plugin_intro_text ) );
+	do_action('pt-ocdi/plugin_intro_text_after');
```

Comment:
since the filter of `pt-ocdi/plugin_intro_text` runs through `wp_kses_post`, so to add `iframe` for youtube tutorial video embed, we can use an action hook here.